modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2518,37 +2517,32 @@ private void _testUpdateObjectEntryExternalReferenceCode()
 			ServiceContextTestUtil.getServiceContext());
 
 		Assert.assertEquals(
-			String.valueOf(objectEntryId2),
-			objectEntry2.getExternalReferenceCode());
+			objectEntry2.getUuid(), objectEntry2.getExternalReferenceCode());
 
-		_objectEntryLocalService.updateObjectEntry(
-			TestPropsValues.getUserId(), objectEntryId2,
-			HashMapBuilder.<String, Serializable>put(
-				"externalReferenceCode", String.valueOf(objectEntryId1)
-			).build(),
-			ServiceContextTestUtil.getServiceContext());
+		ObjectEntry finalObjectEntry2 = objectEntry2;
```

Comment:
Hey @leilanyaragao
Why did you replace the old code for this new one?
I think we need to maintain the old code that calls the update method. In the parameters, we should update the "externalReferenceCode" with the objectEntry1.getExternalReferenceCode();
What do you think?